src/PathResolver/CustomOperationPathResolver.php

```diff
@@ -28,12 +28,12 @@ public function __construct(OperationPathResolverInterface $deferred)
     /**
      * {@inheritdoc}
      */
-    public function resolveOperationPath(string $resourceShortName, array $operation, bool $collection): string
+    public function resolveOperationPath(string $resourceShortName, array $operation, string $operationType): string
```

Comment:
It's a BC break but it can be easily fixed: remove the typehint and support both `string` and `bool`.

Here I would deprecate to pass a `bool`.